src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -322,14 +322,17 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
             _getFeeManagerIssuanceFeeData(_selector);
     }
 
-    /// @dev Calculates the propotion of the fees for the given amount and returns them plus the amount minus the fees
+    /// @dev    Calculates the propotion of the fees for the given amount and returns them plus the amount minus the fees
+    ///         Reverts under the following two conditions:
+    ///             - if (workflow fee + protocol fee) > BPS
+    ///             - if protocol fee amount or workflow fee amounts < 0 given the fee percentage is not zero. This
```

Comment:
Is this really the way we want to go here?
Just reverting? -> I dont have a hard opinion on this, im just curious

Reply:
We can currently only go a few routes:
1. Set a minimum fee to ensure that users can't buy dust amounts, resulting in fee being rounded down. However this also have the consiquence where users pay more fee than they should, if they really want to buy a small amount
2. Set a minimum buy amount to ensure that the amount used for calculating the fee is big enough that the fee amount is not rounded down. This number however needs to be somewhat dynamic to ensure this, and forces the user to commit a certain amount.
3. Revert if the buy/sell amount has been so low that that taking a percentage fee is being rounded down to zero. This would at least protect agains the vulnerbility of repeated dust amount buys with no fee paid

So @0xNuggan and I agreed on going with the last route